src/components/players/playerDetails/playerDetails.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+import Placeholder from './placeholder.js';
+import Stats from './stats';
+
+import styles from './playerDetails.module.scss';
+
+const PlayerDetails = ({ player }) => {
+    if (!player) {
+        return (
+            <div className={styles.container}>
+                <Placeholder />
+            </div>
+        );
+    }
+
+    const {
+      person_id,
+      first_name,
+      last_name,
+      position_full,
+      jersey_number,
+      height_ft,
+      height_in,
+      weight_lbs,
+    } = player;
+
+    return (
+        <div className={styles.container}>
+            <div className={styles.card}>
+                <div className={styles.imageContainer}>
+                    <img src={`https://ak-static.cms.nba.com/wp-content/uploads/headshots/nba/1610612761/2019/260x190/${person_id}.png`} alt="Player Headshot" />
+                </div>
+
+                <div className={styles.imageBorderBottom}></div>
+                <div className={styles.statsContainer}>
+                    <div className={styles.nameContainer}>
+                        <div className={styles.name}>{first_name} {last_name}</div>
+                        <div className={styles.jerseyNumber}>{jersey_number}</div>
+                    </div>
+                    <div className={styles.position}>{position_full}</div>
+                    <div className={styles.details}>{height_ft}-{height_in}, {weight_lbs} lbs</div>
+                </div>
+                <div className={styles.mainStatsContainer}>
+                    <Stats isQuickStats={true} title="Ppg" stat={player.ppg} />
+                    <Stats isQuickStats={true} title="Reb" stat={player.totReb} />
```

Comment:
This is my bad. This value/title should be the `rpg` (rebounds per game):

<img width="130" alt="Screen Shot 2020-03-20 at 5 03 03 PM" src="https://user-images.githubusercontent.com/623755/77206197-c63d4c00-6acc-11ea-851a-960ba56f65aa.png">

If you could change that.